requirements.yml

```diff
@@ -0,0 +1 @@
+- src: dokku_bot.ansible_dokku
```

Comment:
nice - mind updating the README to refer to this instead of the plugin directly as well?

Reply:
@jbothma Potentially we should either have a requirements.yml file per app. It is a separate thing from the ansible playbook itself. Just allows you to more easily perform installation like this:

`ansible-galaxy install -r requirements.yml` before the first time you run the playbook.